kubeless.jsonnet

```diff
@@ -1,7 +1,14 @@
+local k = import "ksonnet.beta.1/k.libsonnet";
 local kube = import "kube.libsonnet";
+local container = k.core.v1.container;
 
 local kubeless = import "vendor/kubeless/kubeless-rbac.jsonnet";
 
+local controllerContainer =
+  container.default("kubeless-controller", "bitnami/kubeless-controller@sha:6547d4e088ad5171dff69da4c9e52e77e6d0b00cf25f8b57c3260de816587266") +
+  container.imagePullPolicy("IfNotPresent");
+
 kubeless {
   ns: kube.Namespace("kubeless"),
+  controller+: {spec+: {template+: {spec+: { containers: [controllerContainer] } } } },
```

Comment:
I think we only need to change the tag, we should be able to do:

```
kubeless {
  controller+: {
    spec+: {
      template+: {
        spec+: {
          containers: kubeless.controller.spec.template.spec.containers[0] + { image: "bitnami/kubeless-controller@sha:6547d4e088ad5171dff69da4c9e52e77e6d0b00cf25f8b57c3260de816587266" }
        }
      }
    }
  }
}
```
untested*

Reply:
The main reason is to make it easier to upgrade when the upstream kubeless.jsonnet changes. E.g. if an env var is added to the container, we won't also need to add it here since we only modify the image.